src/coreclr/tools/aot/ILCompiler/ILCompilerRootCommand.cs

```diff
@@ -20,29 +20,29 @@ internal sealed class ILCompilerRootCommand : RootCommand
         public Option<Dictionary<string, string>> ReferenceFiles { get; } =
             new(new[] { "--reference", "-r" }, result => Helpers.BuildPathDictionay(result.Tokens, false), true, "Reference file(s) for compilation");
         public Option<string> OutputFilePath { get; } =
-            new(new[] { "--out", "-o" }, "Output file path");
+            new(new[] { "--out", "-o" }, result => Helpers.Unquote(result.Tokens), true, "Output file path");
```

Comment:
Can the unquoting be done in `TryReadResponseFile` instead of around every single argument?

Reply:
Can we strip the first and last quote if the string ends with a quote? 
```
if (s.EndsWith('"'))
{
    int firstQuote = s.IndexOf('"');
    if (firstQuote >= 0 && firstQuote < s.Length - 1)
         s = s[..firstQuote] + s[(firstQuote+1)..^1];
}
```
It should be good-enough to maintain compatibility with what we had, without the manual stripping of quotes everywhere.

Reply:
That will work. 👍